package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/Christianlopez4/protractor-workshop-20201#readme",
   "dependencies": {
+    "jasmine-spec-reporter": "^5.0.2",
```

Comment:
please install jasmine and protractor as dev dependencies.

Reply:
i already installed jasmine as dev dependency, but when i tried to do the same with protractor an error was generated
![Captura](https://user-images.githubusercontent.com/64425865/88487739-5c86f000-cf4d-11ea-89b3-9943ec0a982a.PNG)

Reply:
did you run `npm install --save-dev protractor` command?

Reply:
yes, i did it

Reply:
run `npm ci` to reinstall all dependencies.

Reply:
This is strange, the error continues there, but I was able to run the npm test without issue

Reply:
i'll upload it for you to collaborate with me

Reply:
I cloned your repo, ran `npm ci` then `npm run test`, also opened with VSCode and didn't get any error, so I guess that everything is fine.